core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -115,7 +127,21 @@ public int getValue() {
      */
     UnderFileSystem get(String path, Object ufsConf, Configuration configuration) {
       UnderFileSystem cachedFs = null;
-      Key key = new Key(new AlluxioURI(path));
+      Permission perm = Permission.defaults();
+      try {
+        // TODO(chaomin): consider adding a JVM-level constant to distinguish between Alluxio server
+        // and client.
+        String loggerType = configuration.get(Constants.LOGGER_TYPE);
+        if (loggerType.equalsIgnoreCase("MASTER_LOGGER")
```

Comment:
could you add a TODO here to tell whether we are in client JVM or not using some built-in util functions, rather than from logger types?